jupyterlab_git/handlers.py

```diff
@@ -1069,6 +1069,24 @@ async def post(self, path: str = ""):
             self.finish(json.dumps(response))
 
 
+class GitSubModulesHandler(GitHandler):
```

Comment:
Minor nitpick: should `SubModule` be spelled `Submodule` instead?

Looks like this is what the git docs use: https://git-scm.com/book/en/v2/Git-Tools-Submodules

Wondering about this, if we start exposing classes and other interfaces which include the word "submodule" in their name.